index.js

```diff
@@ -66,51 +133,61 @@ const main = async () => {
   }
 
   // Sanity check
-  if (status != 'COMPLETE') {
+  if (status !== 'COMPLETE') {
     throw new Error(`Unhandled scan status "${status}". API response: ${JSON.stringify(findings)}`)
   }
 
+  const findingsList = findings.imageScanFindings.findings
+  const ignoredFindings = findingsList.filter(({ name }) => ignoreList.includes(name))
+
+  if (ignoreList.length !== ignoredFindings.length) {
+    throw new Error(`Ignore list contains CVE IDs that were not returned in the findings result set. They may be invalid or no longer be current vulnerabilities.`)
```

Comment:
This should print the invalid IDs -- users will definitely want them

Reply:
New output:

```
The following CVEs were not found in the result set:
  - CVE-A
  - CVE-C
::error::Ignore list contains CVE IDs that were not returned in the findings result set. They may be invalid or no longer be current vulnerabilities.
```